eng/pipelines/stress-cluster-provision.yml

```diff
@@ -5,10 +5,10 @@ trigger: none
 parameters:
   - name: Environment
     type: string
-    default: test
+    default: pg
```

Comment:
I assume pg stands for playground? Any reason to not spell it out?

Reply:
I think brevity though @ckairen  might have been running into name length issues in few places as well.

Reply:
Brevity from the command line tools perspective, vs. the pipeline perspective where it's just a dropdown.